humble/distribution.yaml

```diff
@@ -5912,6 +5912,24 @@ repositories:
       url: https://github.com/ros-visualization/python_qt_binding.git
       version: humble
     status: maintained
+  qb_device:
+    release:
+      packages:
+      - qb_device
+      - qb_device_bringup
+      - qb_device_driver
+      - qb_device_msgs
+      - qb_device_ros2_control
+      - qb_device_test_controllers
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://bitbucket.org/qbrobotics/qbdevice-ros2-release.git
```

Comment:
While this is allowed for Humble, if you ever want to release this into Rolling this will have to move into the https://github.com/ros2-gbp organization.